ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -73,14 +73,17 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, onAddE
 
     // On live media rebooting the system will actually shut it off
     const isBootIso = conf["Installation System"].type === "BOOT_ISO";
+    const isLiveOS = conf["Installation System"].type === "LIVE_OS";
```

Comment:
Are there more options than liveos and bootiso? If not we just need one variable - not two.

Reply:
There is a BOOTED_OS, but it cant happen in the webui. So i can remove isLiveOS line.